### ISSUE TITLE
Show bookmarks sooner than the 30 minute recent filter

### ISSUE THREAD

**Issue by Mardak** [mozilla/activity-stream#1670]
Split from #1638. The fix for #1493 has allows new pages/bookmarks to show up immediately but potentially up to 15 minutes later. We can special case bookmarks to have them show up sooner -- potentially on the next new tab.

**Comment by Mardak** [comment]
@bryanbell are you still running into this? I've tried testing this, and the bookmarks appear to show up immediately for me. I see this in the console:
```
console.log: activity-streams: Refreshing data for HighlightsFeed because a bookmark was added
```

And the bookmark appears as the first item.

However, while I was testing this. I did run into a situation where only the default highlights were shown, and this was caused by the potential highlight not having any images as per #1635. @sarracini did you run into cases where all highlights had no images?
